### DISCUSSION TITLE
no windows, no menu

### DISCUSSION THREAD

**Discussion by mauricev** [i3/i3#4587] [Q&A] [answered]
I logged in Ubuntu 20.04.1 and chose the i3 window manager. I chose the default configuration. I have a mostly black screen with a "1" icon in the lower left and some machine info printed along the bottom at the right. There are no windows, no menus and no way to get to them or to access a terminal to even print the env variables. What am I missing? It's as if the desktop environment isn't being tied to i3.

**Comment by Airblader** [comment, marked as answer]
This sounds like a default i3 installation. $mod+Enter would bring up a terminal if you have one installed.

Apart from that I would recommend reading the user guide or any other introduction to i3 if you're not familiar with it yet.

> It's as if the desktop environment isn't being tied to i3.

Sorry, j don't understand what you're trying to say here.

**Comment by mauricev** [comment]
OK, I understand now. i3 is bare-bones: there is no desktop environment, so to speak. Anyway, I'm really trying to get it to work with RDP and here the mod key is being ignored for some reason. Has anyone ever got RDP/VNC to work with it?

**Comment by mauricev** [comment]
I am using VNC/RDP from Citrix and it appears that Citrix’s implementation of the client is at fault here.
